ckcp/gitops.sh

```diff
@@ -333,41 +262,22 @@ install_triggers_interceptors(){
     KUBECONFIG=$KUBECONFIG_KCP oc create secret generic kcp-kubeconfig -n tekton-pipelines --from-file "$KUBECONFIG_KCP" >/dev/null
   fi
 
-  #############################################################################
-  # Install triggers interceptors
-  #############################################################################
-  echo -n "  - $APP application: "
-  if ! oc get apps -n openshift-gitops "$APP" >/dev/null 2>&1; then
-    oc apply -f "$GITOPS_DIR/$APP.yaml" --wait >/dev/null
-  fi
-  argocd app wait $APP >/dev/null
-  echo "OK"
+  install_app triggers-interceptors
 }
 
 install_triggers_controller(){
-  APP="triggers-controller"
-  export KUBECONFIG="$KUBECONFIG_CLUSTER"
-
   #############################################################################
   # Create kcp-kubeconfig secret for triggers controller
   #############################################################################
-  if ! oc get namespace triggers >/dev/null 2>&1; then
-    oc create namespace triggers >/dev/null
-  fi
-  if ! oc get secret ckcp-kubeconfig -n triggers >/dev/null 2>&1; then
+  oc create namespace triggers -o yaml --dry-run=client | oc apply -f- &>/dev/null
+  if ! oc get secret ckcp-kubeconfig -n triggers &>/dev/null; then
```

Comment:
if loop can be replaced

oc create secret generic ckcp-kubeconfig -n triggers --from-file "$KUBECONFIG_KCP" --dry-run=client -o yaml | oc apply -f - &>/dev/null